Makefile

```diff
@@ -72,6 +76,9 @@ win32: create_archive_dir man git_export
 	cd "$(EXPORT_DIR)"; $(PYTHON_EXE) setup.py bdist_wininst --user-access-control force \
 		--dist-dir "$(ARCHIVE_DIR)" $(DISTUTILS_PLAT_NAME)
 
+deb: git_export
+	cd "$(EXPORT_DIR)"; dpkg-buildpackage -us -uc
```

Comment:
This omits uncommitted local changes from the debian package, including (if i understand this correctly) the updated debian/changelog.